htdocs/web_portal/views/downtime/add_downtime.php

```diff
@@ -294,6 +305,22 @@ function updateStartEndTimesInUtc(){
         }
    }
 
+    function hasSitesWithSingleTimezones()
```

Comment:
this method should refer to selecting multiple sites, than sites being in the same timezone.

Reply:
I would like to go with hasSitesWithSingleTimezones() because its about user selecting site time zone (Which we are not supporting) option along with multiple sites.

Reply:
the check here is that multiple sites have been selected, not that wether those sites share a timezone.

Reply:
`hasMultipleSitesWithSiteTimezones()`